src/services/insights/readme.md

```diff
@@ -0,0 +1,97 @@
+# Insights API
+[Official API docs](http://localhost:3030/docs/#/insights)
+### Allowed HTTPs requests:
+``GET ``
+### Description Of Usual Server Responses:
+200 `OK`
+401 `Unauthorized`
+## Monthly Users 
+##### One sentence explanation here
+`/insgihts/monthlyUsers`
+`GET`
+expected result:
+```
+{
+thisWeek: String, // eg: "42"
+lastWeek: String
```

Comment:
I believe you meant thisMonth not thisWeek

Reply:
done. updated in next push